### PR TITLE
Add read/write -index_prefix

### PR DESCRIPTION
**Pull request by davidvgalbraith**
This specifies the prefix for the index names you want to
read from. The suffix is the day of the points in the index.
If you don't specify a prefix, assumes you want logstash-.
This default can be overwritten in each configured backend
by adding an index_prefix key.

@demmer @VladVega

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 var _ = require('underscore');
 
+var DEFAULT_PREFIXES;
+var DEFAULT_DEFAULT_PREFIX = 'logstash-';
```
Comment (VladVega) [inline lib/utils.js]: `DEFAULT_DEFAULT`?

Comment (VladVega) [inline lib/utils.js]: Consider `LOGSTASH_PREFIX?

Comment (davidvgalbraith) [inline lib/utils.js]: I considered it! I like `DEFAULT_DEFAULT_PREFIX`, though. Kinda rolls off the tongue, you know?

**Comment by davidvgalbraith** [comment]
Slack'd +1 from @VladVega.

**Comment by henridf** [comment]
no README update?

**Pull request merged**